tests/examples/minimal/package.json

```diff
@@ -6,7 +6,8 @@
         "build": "tsc"
     },
     "devDependencies": {
-        "typescript": "^3.0.0"
+        "typescript": "^3.0.0",
+        "@types/node": "^11.0.0"
```

Comment:
Something is causing proto-loader to get imported with these new changes which causes a conflict with types/node. Looks like we do generally set this in devDeps these days, this was a _very_ old test. But would be good if we could work out what's triggering this proto-loader import that trips this up.

Reply:
Somewhat concerning that this could impact existing user programs if they don't already have `@types/node` defined, but we've specified `@types/node` in our `pulumi new` templates forever, so I'm guessing it's quite unlikely anyone would hit this.

What is the actual error without it?

Reply:
`Conflicting definitions for 'node' found`, but I only saw it in this one test which is like 4/5 years old.